src/serverUtils.js

```diff
@@ -132,19 +136,23 @@ const BaseServer = class {
         if (!(msg[1] in this.peer_list)) {
           const peer = msg[1].split(':');
           this.addPeer(peer[0], peer[1]);
+          console.log("hello")
         }
       }
-    } else if ((type === 2) && (msg.length === 1) && (msg[0] === 'ok')) {
+    } 
+    else if ((type === 2) && (msg.length === 1) && (msg[0] === 'ok')) {
```

Comment:
how would you feel about making `type` a string descring `tcp`, `ws`, `peer`, etc?

I can make the change, just wishing for our long-term readability. My working memory is much smaller these days, so it's my guide.

Reply:
Yea sure that works, I'd rather prefer a enum kind of thing, like vars TCP, WS, PEER with specific values attached. I think that'll be more readable and reduce unwanted confusion with case sensitive string compression.

Reply:
I will leave this for next PR, it works fine the way it is now, and you already use type numbers elsewhere.